### PR TITLE
Add @sarahbenmessaoud

### PR DESCRIPTION
**Pull request by sarahbenmessaoud**

### CODE REVIEW

File: site/class-2-debug/index.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+html {
+    font-family: Roboto, -apple-system, BlinkMacSystemFont, Helvetica Neue, Segoe UI, Oxygen, Ubuntu, Cantarell, Open Sans, sans-serif;
+    max-width: 100%;
+}
+body {
+    margin: 0;
+}
+svg {
+    width: 100w;
```
Comment (wooorm) [inline site/class-2-debug/index.css]: w -> vw

File: site/class-2-debug/index.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    height: 100h;
```
Comment (wooorm) [inline site/class-2-debug/index.css]: h -> vh

File: site/class-2-debug/index.css (hunk 1 above)
Comment (sarahbenmessaoud) [reply site/class-2-debug/index.css]: Oeh heb deze 2 niet gezien dankje!

**Comment by wooorm** [comment]
You didn’t upload in the right directory 😢

Your work is late and I unfortunately don’t have to time to review student work passed the due date.

Feel free to upload more new work, or this work, if it’s uploaded in the right place and correct though, you won’t get points for it, but I’d of course love to see it!

**Pull request closed without merging**